### PR TITLE
Fixed snyk issues, fixed test files variable name mismatches and dele…

### PR DESCRIPTION
**Pull request by sunil-lakshman**
…ted unused github action file

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -59,6 +59,6 @@ distlib~=0.3.6
 Empty~=0.4.4
 cachetools~=5.3.0
 tomlkit~=0.11.8
```
Comment (ishaileshmishra) [inline requirements.txt:61]: Do we need these all libraries for the SDK to run @sunil-lakshman

File: setup.py [hunk 2]
```diff
@@ -25,7 +25,7 @@
     type="process",
     created="09 Jun 2020",
     keywords="contentstack-python",
-    version="1.8.0",
+    version="1.8.1",
     author="Contentstack",
     author_email="shailesh.mishra@contentstack.com",
```
Comment (ishaileshmishra) [inline setup.py:30]: author_email="shailesh.mishra@contentstack.com", "sunil.lakshman@contentstack.com"

**Review by ishaileshmishra — CHANGES_REQUESTED** [review]
Need to check all the dependencies are really used to run the sdk or few of them are redundant ? @sunil-lakshman

File: requirements.txt (hunk 1 above)
Comment (sunil-lakshman) [reply requirements.txt:61]: We are using above packages @ishaileshmishra 
1. **Empty** - is a light wrapper around Flask which adds some sensitive defaults to your project out of the box.
2. **tomlkit**  -  It includes a parser that preserves all comments, indentations, whitespace and internal element ordering, and makes them accessible and editable via an intuitive API.
3. **cachetools** - For our purpose, a cache is a key-value store, aka a dictionary, possibly with some constraints on keys (type, size) and values (size, serialization). Dependency for **logging** package.

Comment (ishaileshmishra) [reply requirements.txt:61]: @sunil-lakshman What if you remove those and run tests. Does tests fail ?

Comment (sunil-lakshman) [reply requirements.txt:61]: yes @ishaileshmishra

Comment (ishaileshmishra) [reply requirements.txt:61]: I doubt about Empty and cachetools. Are they really required ?

Comment (sunil-lakshman) [reply requirements.txt:61]: Getting above error if i remove Empty from code.

Comment (ishaileshmishra) [reply requirements.txt:61]: okay got it, what if we remove cachetools only ?

**Review by ishaileshmishra — APPROVED** [review]
approved

**Pull request merged**